trunk/SUAVE/Input_Output/OpenVSP/vsp_wing.py

```diff
@@ -113,7 +116,7 @@ def read_vsp_wing(wing_id, units_type='SI', write_airfoil_file=True, use_scaling
         wing.tag = 'winggeom'
     
     if use_scaling:
-        scaling       = vsp.GetParmVal(fuselage_id, 'Scale', 'XForm')  
+        scaling       = vsp.GetParmVal(wing_id, 'Scale', 'XForm')
```

Comment:
Thanks for catching this. I realized this yesterday right after the SUAVE meeting